wolfcrypt/src/dh.c

```diff
@@ -1388,20 +1406,132 @@ static int wc_DhGenerateKeyPair_Async(DhKey* key, WC_RNG* rng,
 #endif /* WOLFSSL_ASYNC_CRYPT && WC_ASYNC_ENABLE_DH */
 
 
+/* Performs a Pairwise Consistency Test on an FFC key pair. */
```

Comment:
Recommend moving this down above `wc_DhCheckKeyPair` to reduce line count changes and make diff easier.

Reply:
Done.